scripts/o80_print_data.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+"""Connect to a running o80 Standalone instance and print the received data.
```

Comment:
a more precise wording: it connects to the share memory to which the o80 backend operated by the standalone writes to.

Reply:
Thanks, I rewrote the docstring accordingly.